### PR TITLE
feat(grpc): grpc keepalive configuration

### PR DESCRIPTION
**Pull request by erka**
I believe it's all that should be done in Flipt sources. All other is helm related

refs #2428

### CODE REVIEW

File: internal/config/testdata/server/grpc_keepalive.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#   memory:
+#     eviction_interval: 5m # Evict Expired Items Every 5m
+
+server:
+#   protocol: https
+#   host: 0.0.0.0
+#   https_port: 443
+#   http_port: 8080
+#   grpc_port: 9000
+    grpc_max_connection_idle: 1h
```
Comment (markphelps) [inline internal/config/testdata/server/grpc_keepalive.yml:25]:
```suggestion
    grpc_conn_max_idle_time: 1h
```
 To match similar redis setting

File: internal/config/testdata/server/grpc_keepalive.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    grpc_max_connection_age: 30s
```
Comment (markphelps) [inline internal/config/testdata/server/grpc_keepalive.yml:26]:
```suggestion
    grpc_conn_max_age: 30s
```
Wdyt? Less to type and matches existing patterns

File: internal/config/testdata/server/grpc_keepalive.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    grpc_max_connection_age_grace: 10s
```
Comment (markphelps) [inline internal/config/testdata/server/grpc_keepalive.yml:27]:
```suggestion
    grpc_conn_max_age_grace: 10s
```

**Review by markphelps — COMMENTED** [review]
Thanks @erka !! Some minor rename suggestions for consistency w other settings.

**Review by GeorgeMac — APPROVED** [review]
Noice 💪

**Pull request merged**